cookbook/controller/upload_file.rst

```diff
@@ -146,16 +159,26 @@ Finally, you need to update the code of the controller that handles the form::
         }
     }
 
+Now, create the ``brochures_directory`` parameter that was used in the
+controller to specify the directory in which the brochures should be stored:
+
+.. code-block:: yaml
+
+    # app/config/parameters.yml
+    parameters:
+        # ...
+        brochures_directory: '%kernel.root_dir%/../web/uploads/brochures'
```

Comment:
This makes more sense in `config.yml` - it's not a value that would normally need to be changed on a machine-by-machine basis, and devs wouldn't/shouldn't need to worry about configuring it.